benchmarks/nuget.config

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file isn't needed, it should all be coming from the root Directory.Build.props:
https://github.com/terrafx/terrafx.interop.mimalloc/blob/main/Directory.Build.props#L68-L71